hack/build_etcd.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+if [  $# -gt 0 ]
+    then
+    ETCD_VERSION="$1";
+    else
+    ETCD_VERSION="master";
+fi
+
+echo "Using ETCD version $ETCD_VERSION"
+
+mkdir /hack
```

Comment:
I don't know where place etcd code becouse if I put it in the working directory then `go build ./...` invoked by Makefile fails due it try to build etcd